src/style/components/app-notification.less

```diff
@@ -0,0 +1,40 @@
+.app-notification {
```

Comment:
Pretty much the same styles as before, see the deleted file `src/script/components/AppNotification/AppNotification.styles.ts`